src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,49 @@
 package core.basesyntax;
 
+import java.io.*;
+import java.util.HashMap;
+import java.util.Map;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
```

Comment:
The access modifier for 'getStatistic' should likely be 'private' if it is only intended to be used within this class.